config/environments/development.rb

```diff
@@ -33,8 +33,21 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
+
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+  address: 'smtp.gmail.com',
```

Comment:
I think we should/could expect SMTP to be active on the server or in the infrastructure rather than use gmail (I understand this was just for testing).. 

I'd like to see the development, production, and staging configurations default to using SMTP local to the application (assume that the server has smtp) and those configuration values tie to ENV but with localhost defaults.